### PR TITLE
Test/fix master tests

### PR DESCRIPTION
**Pull request by molnosikristof**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -87,6 +87,9 @@ jobs:
               git config --global user.name "purpose-bot"
               git config --global push.default simple
               npm version $RELEASE_TYPE --commit-hooks=false
+              #git add package.json package-lock.json
```
Comment (oroce) [inline .circleci/config.yml]: can you remove this? it's not even in the master

Comment (oroce) [reply .circleci/config.yml]: and the next 2 lines

File: .circleci/config.yml [hunk 2]
```diff
@@ -119,7 +122,7 @@ workflows:
   nightly_build:
     triggers:
       - schedule:
-          cron: "00 16 * * *"
+          cron: "30 16 * * *"
```
Comment (oroce) [inline .circleci/config.yml]: why is this changed?

Comment (molnosikristof) [reply .circleci/config.yml]: no reason, this is caused by versioning.
